### PR TITLE
Added new species Blurbs + Misc Eris removals.

### PR DESCRIPTION
**Pull request by SirSiggles**
## About The Pull Request
Removed some minor Eris leftovers and added new blurbs to Skrell, Humans, Unathi and Vulps. 

## Why It's Good For The Game
The Blurbs needed a retouching up because they weren't very good. (Especially Vulps! :honk:) and I got some Eris stuff I missed.

## Changelog
:cl:
del: Removed Eris mentions.
tweak: Rewrote Species Blurbs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 1]
```diff
@@ -53,11 +53,11 @@
 
 //	economic_modifier = 7
 
-	blurb = "A heavily reptillian species, Unathi hail from the \
-	Uuosa-Eso system, which roughly translates to 'burning mother'.<br/><br/>Coming from a harsh, inhospitable \
-	planet, they mostly hold ideals of honesty, virtue, proficiency and bravery above all \
-	else, frequently even their own lives. They prefer warmer temperatures than most species and \
-	their native tongue is a heavy hissing laungage called Sinta'Unathi."
+	blurb = "A reptillian species from the Uuosa-Eso system that speaks the hissing Sinta'Unathi language \
+	most Unathi are guided by a complex Ethnoreligion called the Unity,, \
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/species/station/station.dm:57]: Mild typo here, your comma key double-struck

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 2]
```diff
@@ -53,11 +53,11 @@
...
+	which stresses perfection of one's craft, honesty and bravery \
+	discovered less then a generation ago by SolGov,\
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/species/station/station.dm:59]: "...which stresses perfection of one's craft, honesty and bravery discovered less than a generation ago..."?

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 3]
```diff
@@ -167,11 +167,11 @@
...
-	blurb = "The Tajaran are a mammalian species resembling roughly felines, hailing from Meralar in the Rarkajar system. \
-	While reaching to the stars independently from outside influences, the humans engaged them in peaceful trade contact \
-	and have accelerated the fledgling culture into the interstellar age. Their history is full of war and highly fractious \
-	governments, something that permeates even to today's times. They prefer colder, tundra-like climates, much like their \
-	home worlds and speak a variety of languages, especially Siik and Akhani."
+	blurb = "The Tajaran are a feliform species, from Meralar in the Rarkajar system \
+	with a rich cultural history influenced by three great ethnic groups \
+	the Hharar who are the most populous and with noble lineages. The great explorers of the Njarir'Akhan and the barbarians of the Zhan-Khazan. \
+	Unlike many other species, the Tajaran homeworld is still not fully united and \
+	even now Tajara speak a variety of languages, including Siik and Akhani."
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/species/station/station.dm:174]: The frst two sentences on this don't read very well. 

> The Tajaran are a feliform species, from Meralar in the Rarkajar system with a rich cultural history influenced by three great ethnic groups the Hharar who are the most populous and with noble lineages. The great explorers of the Njarir'Akhan and the barbarians of the Zhan-Khazan. ..."

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 4]
```diff
@@ -249,11 +249,11 @@
...
-	blurb = "Vulpkanin (Commonly called Vulps or Vulpa for short) are a race of Canids from the planet of Altam in the Binary Vazzend system. \
-	Vulpkanin have a long history of religious and philosophical conflicts, ended by the current government the Vulpkanin nation,\
-	 an isolationist nation led by the ideas of utilitarianism with harsh criminal policies,\
-	including the generational exile of criminal and dissidents. However, a close working relationship with NanoTrasen means that while most Vulp characters are descendants or exiles\
-	themselves from Vulpkanin Space, some characters are citizens of the Vulp nation."
+	blurb = "Vulpkanin are a race of Canid-likes from the Vazzend system with a long history of. \
+	of religious and philosophical conflicts, ended by the current unifiying body, the Vulpkanin Nation \
+	an isolationist and totalitarian stellar state led by  ideas similar to Utilitarianism and with harsh criminal policies,\
+	including the generational exile of criminals and dissidents. Meaning there is a large exodus of Vulpkanins to foreign space\
+	though there remains a small community of temporary expats, still loyal to their home."
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm:256]: This reads like a bit of a word salad. 

> ...from the Vazzend system with a long history of. of religious and philosophical conflicts...

> ...the Vulpkanin Nation an isolationist and totalitarian stellar state led by ideas similar to Utilitarianism and with harsh cominal policiesincluding the generational exile...

>...there is a large exodus of Vulpkanins to foreign spacethough there remains a small community...

File: code/modules/mob/living/carbon/human/species/station/station.dm (hunk 2 above)
Comment (SirSiggles) [reply code/modules/mob/living/carbon/human/species/station/station.dm:59]: My Bad. Yeah.

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm (hunk 4 above)
Comment (SirSiggles) [reply code/modules/mob/living/carbon/human/species/station/station_vr.dm:256]: Fixing it. "Cominal" Jeez! Whoever wrote that one sure is a hack fraud!

Comment (EvilJackCarver) [reply code/modules/mob/living/carbon/human/species/station/station_vr.dm:256]: I blame laptop keyboards.

**Comment by SirSiggles** [comment]
Should be better hopefully now!

**Pull request merged**